migrations/versions/d2ae8e54b628_.py

```diff
@@ -0,0 +1,69 @@
+"""shorten columns used in UNIQUE constraints in periodictasklastrun, periodictaskoption
+
+Revision ID: d2ae8e54b628
+Revises: 1a0710df148b
+Create Date: 2018-10-24 11:17:40.279312
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'd2ae8e54b628'
+down_revision = '1a0710df148b'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # First, try to create the table with the shorter field sizes.
+    # This will fail if the tables already exist (which will be
+    # the case in most installations)
+    try:
+        op.create_table('periodictasklastrun',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('periodictask_id', sa.Integer(), nullable=True),
+        sa.Column('node', sa.Unicode(length=255), nullable=False),
+        sa.Column('timestamp', sa.DateTime(), nullable=False),
+        sa.ForeignKeyConstraint(['periodictask_id'], ['periodictask.id'], ),
+        sa.PrimaryKeyConstraint('id'),
+        sa.UniqueConstraint('periodictask_id', 'node', name='ptlrix_1'),
+        mysql_row_format='DYNAMIC'
+        )
+        op.create_table('periodictaskoption',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('periodictask_id', sa.Integer(), nullable=True),
+        sa.Column('key', sa.Unicode(length=255), nullable=False),
+        sa.Column('value', sa.Unicode(length=2000), nullable=True),
+        sa.ForeignKeyConstraint(['periodictask_id'], ['periodictask.id'], ),
+        sa.PrimaryKeyConstraint('id'),
+        sa.UniqueConstraint('periodictask_id', 'key', name='ptoix_1'),
+        mysql_row_format='DYNAMIC'
+        )
+        print('Successfully created shortened periodictasklastrun and periodictaskoption.')
```

Comment:
Usually we do not do this print. - till now.
But it looks nice.

Reply:
Ah, I thought this would be nice because every installation would run into one of the ``except`` blocks -- so it would be nice to have a message that all is fine :-)